src/symbolic/interface.jl

```diff
@@ -0,0 +1,78 @@
+export symbolize
```

Comment:
I like the `symbolize` function. But I'd argue that it makes for better documentation to have the titles and examples in a normal doc page that documenter runs example blocks, rather than inside the docstring. So, we still would have 2 docstirngs for the two methods, that are expanded in documenter `@docs`, but the examples would be runnable code in an `@example` block. It isn't a big difference honestly, so feel free to not do it if you don't want to!

Reply:
Good point @Datseris, and to some extent I agree with you. However, I'm not prioritizing making large structural changes to the documentation at the moment. But it is something to think about when I have more time on my hands.

For the bigger picture:
As you may notice, in this PR, I actually use the `jldoctest` for the new symbolization function signature. Then, one can locally run `doctest(Entropies)` to check that examples are up-to-date and correct. It is also possible to include the `doctest(Entropies)` call in the `runtests.jl` file, so that the documentation is tested too during CI. I want to start using this workflow for all CausalityTools and friends packages.

For me, it makes sense to have a few simple `jldoctest` blocks in the actual docstring of a function (like I did for `symbolize(x, s::GaussianSymbolization)` here), then expand on those simple examples (if necessary) in the documentation pages. For users like me, who prefer to just `?whatever_function` in the command line, it is a bit cumbersome to have to search for the online docs to get a simple example to run. People have different preferences, though, so I see the upside of both approaches.

Reply:
That's okay with me.